feat: pass stdin when executing subcommand

moreal opened the pull request:
This pull request makes `StandaloneSubcommand.execSync` to support `stdin` input.

longfin commented on src/main/headless/validation.ts:
```diff
@@ -4,7 +4,7 @@ import { StandaloneSubcommand } from "./subcommand";
   public isValidPrivateKey(privateKeyHex: string): boolean {
     try {
-      this.execSync("validation", "private-key", privateKeyHex);
+      this.execSync(["validation", "private-key", privateKeyHex]);
```
Is it intended?

```suggestion
      this.execSync(["validation", "private-key"], privateKeyHex);
```

moreal replied:
Yes, it was intended but the code you suggested is correct and right. But `NineChronicles.Headless.Executable validation private-key` doesn't support stdin check. It should be implemented in **planetairum/NineChronicles.Headless** first.

area363 reviewed (approved):
Branch update required!

moreal commented:
> 
> 
> Branch update required!

@area363 I updated this branch on the latest of the target branch and I updated one more file because the `src/main/headless/apv` command was introduced. Could you review it also please? 🙏🏻

area363 commented:
E2E-Test seems to fail 😢
